commit_trees.go

```diff
@@ -37,49 +37,69 @@ func (commitTreesTable) String() string {
 	return printTable(CommitTreesTableName, CommitTreesSchema)
 }
 
-func (commitTreesTable) Resolved() bool { return true }
-
 func (commitTreesTable) Name() string { return CommitTreesTableName }
 
 func (commitTreesTable) Schema() sql.Schema { return CommitTreesSchema }
 
-func (t *commitTreesTable) TransformUp(f sql.TransformNodeFunc) (sql.Node, error) {
-	return f(t)
+func (t *commitTreesTable) WithFilters(filters []sql.Expression) sql.Table {
+	nt := *t
+	nt.filters = filters
+	return &nt
 }
 
-func (t *commitTreesTable) TransformExpressionsUp(f sql.TransformExprFunc) (sql.Node, error) {
-	return t, nil
+func (t *commitTreesTable) WithProjection(colNames []string) sql.Table {
+	return t
```

Comment:
doesn't accept this table pushdowns of projections?

Reply:
same answer